packages/generators/utils/checkDefaults.ts

```diff
@@ -0,0 +1,23 @@
+/**
+ *
+ * Utility function to check if the user config is same as default.
+ *
+ * @param	{Object[]} initConfig 	- config generated by the user
+ * @returns {Boolean} - Whether the default condif is used or not
+ */
+
+export default function checkDefaults(initConfig: Object[]) {
+  // This is the default prompts selections generated on picking default options.
+  const defaultConfig = {
+		babelConfirm: true,
```

Comment:
Do we need to check all prompts or just the entry and output would do? 🤔

Reply:
@evenstensberg @ematipico @dhruvdutt is this the correct approach?

Reply:
All of them. If the user would decide to change **only** the styling config, we would need to generate the config file, even though `output` and `entry` are untouched. 

On the other hand, the check can be done at the very end. No need to check it every time, correct?

Reply:
> All of them. If the user would decide to change only the styling config, we would need to generate the config file, even though output and entry are untouched.

Yeah, works this way rn 👍 

> On the other hand, the check can be done at the very end. No need to check it every time, correct?

yeah it is checking at the end, I'm just storing the user's pick and comparing it at the end before the project scaffolding starts. Did you face any issue while doing a test run? 🤔

Reply:
Oh I see it now! Sorry, I overlooked the overall logic! Looks good to me